pip/update.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+import os
+
+update_list = []
+p = os.popen('sudo -H pip freeze --local',"r")
+while 1:
+    line = p.readline()
+    if not line: break
```

Comment:
I wouldn't worry too much, as it's unlikely we'd accept this PR as a standalone utility script in the pip source tree anyway (and there are a few discussions about a "proper" subcommand, such as #3744 and #3194 - none have yet been accepted because there are some hard issues to be resolved).